version-stable-r-development.Rmd

```diff
@@ -137,56 +141,76 @@ run_rstudio_ver() {
     -e DISPLAY=$DISPLAY \
     -v /tmp/.X11-unix:/tmp/.X11-unix:ro \
-    --name $CONTANER_NAME \
+    --name $CONTAINER_NAME \
     $BASE_IMAGE &&
   # R and RStudio are not getting the DISPLAY environment variable
-  docker exec $CONTANER_NAME bash -c \
+  docker exec $CONTAINER_NAME bash -c \
     'echo "DISPLAY=${DISPLAY}" >> /usr/local/lib/R/etc/Renviron' &&
   # Install Meld
-  docker exec $CONTANER_NAME bash -c \
+  docker exec $CONTAINER_NAME bash -c \
     'apt-get update && apt-get install -y --no-install-recommends meld dbus-x11' &&
-  echo "RStudio running in container "$CONTANER_NAME" on port "$PORT &&
+  echo "RStudio running in container "$CONTAINER_NAME" on port "$PORT &&
   echo "visit http://localhost:"$PORT
 }
 ```
 
-which you can re-use as compact command for any R version as follows
+which you can re-use as compact command for any R version:
 ```{bash run_rstudio_ver-use}
-run_rstudio_ver 3.6.1 RStudioProjects
+run_rstudio_ver 4.4.1 RStudioProjects
 ```
 
 Note that `--restart=always` specifies that the container should stay up and restart
 itself after stopping, e.g. upon machine reboot or docker upgrade, so that it is
 always available. Still, you can explicitly stop the running container with
 ```{bash stop}
-docker stop rstudio_3.6.1
+docker stop rstudio_4.4.1
 ```
 Alternatively, you can omit `--restart=always` and explicitly start the
 container whenever needed with
 ```{bash start}
-docker start rstudio_3.6.1
+docker start rstudio_4.4.1
 ```
 
-Note that `start`/`stop` operations do not affect the persistence of files
-created outside the shared location, including global RStudio options such as
-dark theme, diagnostic, etc. (set via _Tools > Global Options..._). On the other
-hand, these files and settings do ~~not~~ (see above) persist removing the
-container (`docker rm`, see below) .
+Note that `start`/`stop` operations do not affect the persistence of any files
+created in rstudio while the container is running.
+However if the container is _removed_, files created outside of mounted volumes 
+do **not** persist (`docker rm`, see below).
+This is why we use a mounted volume for the `~/.config/rstudio` directory.
+
+### Using `podman` instead of `docker`
 
+`podman` can be used instead of `docker` to run the above commands. In that case, the active user will be _root_, 
+which is then mapped to the user id of the user running the container when writing files to the shared volume.
+Because of this, RStudio does not set the desired home directory as the initial working directory. To correct this, add
+```
+echo '{ "initial_working_directory": "/home/rstudio" }' > $HOME/.rstudio-docker/$R_VER/rstudio-prefs.json &&
+```
+After `mkdir -p $HOME/.rstudio-docker/$R_VER` in the `run_rstudio_ver` function.
+If you run into issues, this rocker [pull request](https://github.com/rocker-org/rocker-versioned2/pull/636) 
+about rootless container support may help.
+
+### Supported R versions
```

Comment:
I propose making a note about the supported R versions at the beginning of the chapter, where we mention that we use the r-ver stack, similarly to the note in https://rocker-project.org/images/versioned/r-ver.html

> This document is for R 4.0.0 >= images. Please check the [rocker-org/rocker-versioned](https://github.com/rocker-org/rocker-versioned) repository for R <= 3.6.3 images.

w/o bothering about any instruction specific to the older versions.